tests/unittest_scoped_nodes.py

```diff
@@ -1678,10 +1678,8 @@ def __init__(self):
                 "FinalClass",
                 "ClassB",
                 "MixinB",
-                "",
                 "ClassA",
                 "MixinA",
-                "",
```

Comment:
This was added in 629c92db2dc1b016f4bf47645c95c42e65fd3bd6

Reply:
I'll try to dig in here. It's the call to `clear_inference_tip_cache` in the new test that is causing test isolation problems.

Reply:
👍

Reply:
Fix is in #1528 (blocked by another issue)